easybuild/scripts/bootstrap_eb.py

```diff
@@ -224,6 +236,9 @@ def stage0(tmpdir):
     from distribute_setup import main as distribute_setup_main
     orig_sys_argv = sys.argv[:]  # make a copy
     sys.argv.append('--prefix=%s' % tmpdir)
+    # We download a custom version of distribute: it uses a newer version of markerlib to avoid a bug (#1099)
+    # url should become: http://raw.githubusercontent.com/hpcugent/easybuild-framework/master/easybuild/scripts/
+    sys.argv.append('--download-base=http://users.ugent.be/~wpoelman/')
```

Comment:
can you also clarify things a bit in which way the distribute we download here is 'custom'? how was it patched compared to the upstream version?